src/components/swap/SwapRoute.tsx

```diff
@@ -22,16 +23,18 @@ const Separator = styled.div`
   width: 100%;
 `
 
-const V2_DEFAULT_FEE_TIER = 3000
+const V2_DEFAULT_FEE_TIER = FeeAmount.MEDIUM
```

Comment:
small nit i still am not a fan of low medium high - but for this release seems okay

Reply:
lol yeah i dislike it too especially that we have a lowest tier now.. :/ but definitely a big change